data/content_scripts/posting_process/context_menu.js

```diff
@@ -35,7 +35,7 @@ postingProcess.menuClickHandler = function(node, data) {
     text: selectionText,
     pageURL: window.location.href,
   };
-  self.postMessage(info);
+  self.postMessage({name: "startPostingProcess", content: info});
```

Comment:
> Could you have the onMessage handler forward the message to the context_messenger?

The Privly.message framework doesn't support that, of course we could make some changes but IMO, that would defeat the purpose of the context messenger.

@SummerWish Any ideas ?